UnityProject/Assets/Scripts/Lifecycle/Spawn.cs

```diff
@@ -211,7 +222,11 @@ public static SpawnResult ClientPrefab(GameObject prefab, Vector3? worldPosition
 	public static SpawnResult ServerPrefab(string prefabName, Vector3? worldPosition = null, Transform parent = null,
 		Quaternion? rotation = null, int count = 1, float? scatterRadius = null, bool cancelIfImpassable = false)
 	{
-		return Server(SpawnInfo.Prefab(prefabName, worldPosition, parent, rotation, count, scatterRadius, cancelIfImpassable));
+		return Server(
+			SpawnInfo.Spawnable(
+				SpawnablePrefab.For(prefabName),
+				SpawnDestination.At(worldPosition, parent, rotation),
+				count, scatterRadius));
```

Comment:
`cancelIfImpassable` flag might be ignored here, unless I misunderstood something